pkg/kubectl/cmd/diff.go

```diff
@@ -113,13 +80,12 @@ func NewCmdDiff(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.C
 	cmd := &cobra.Command{
 		Use: "diff -f FILENAME",
 		DisableFlagsInUseLine: true,
-		Short:   i18n.T("Diff different versions of configurations"),
+		Short:   i18n.T("Diff live version against would-be applied version"),
 		Long:    diffLong,
 		Example: diffExample,
 		Run: func(cmd *cobra.Command, args []string) {
-			from, to, err := parseDiffArguments(args)
-			cmdutil.CheckErr(err)
-			cmdutil.CheckErr(RunDiff(f, &diff, &options, from, to))
+			cmdutil.CheckErr(checkDiffArgs(cmd, args))
```

Comment:
In the next iteration please update this command to follow the patterns used in other commands. Such as having a dedicated `*Options` struct and 3 methods `Complete`, `Validate` and `Run` working on that struct.